src/components/StatusBarsForNormalization.jsx

```diff
@@ -0,0 +1,41 @@
+import React from 'react'
+import UnAssigned from '../logos/UnAssigned.svg';
+import Assigned from '../logos/Assigned.svg';
+import InProgress from '../logos/InProgress.svg';
+import Completed from '../logos/Completed.svg';
+import StatusBarTail from './StatusBarTail';
+
+function StatusBarsForNormalization() {
+    return (
+       <div className="grid grid-cols-4 gap-2">
+          <div>
+             <StatusBarTail
```

Comment:
Why are u created components with `tail` suffix. 
What is this tail ?

Reply:
I have renamed it

Reply:
Then why it is still the same ?